Cookbook/Technical-Documents/PullAssigners.md

```diff
@@ -2,72 +2,64 @@
+1. Head to the organisation's [home page](https://github.com/babylonhealth).
+2. Go to the "Teams" tab and search for your team ([e.g. here for `iOS-Devs`](https://github.com/orgs/babylonhealth/teams/ios-devs)).
+3. Go to the "Settings" page (to access / edit your team's settings you'll need to have the `Maintainer` team role).
+4. In "Settings" go to "Code review assignment".
+5. Tick "Enable auto assignment" and configure the settings as desired. 
 
-Ask `#devops` to go to the GitHub organization's settings and under the "Installed GitHub Pages" [here](https://github.com/organizations/babylonhealth/settings/installations).
+**Note:** You'll probably want to tick "If assigning team members, don't notify the entire team" to prevent the entire team getting notifications for every PR, even if they haven't been individually assigned to it.
 
-From there they should be able to edit the "PullAssigner" app's settings and add your new repo to the list of repositories the PullAssigner GitHub app has access to
+Please see [GitHub's documentation](https://help.github.com/en/github/setting-up-and-managing-organizations-and-teams/managing-code-review-assignment-for-your-team) for more in-depth details around configuring these settings.
 
-## The `CODEOWNERS` file
+#### In summary
 
-Now we need to ensure that the proxy team (e.g. `iOS-PullAssigners`) is automatically affected as reviewer on all new PRs.
+Now that we have a team of reviewers, your project has a `CODEOWNERS` file and GitHub code review assignment is enabled, the following will happen when you create a PR:
 
-For this, you just need to create a `.github/CODEOWNERS` file in your repo with the following content (if you have a `CODEOWNERS` file already at the root of your repository, we advise to move it inside a `.github/` directory to clean up your repo root)
+1. The team defined in `CODEOWNERS` will be automatically assigned as a reviewer.
+2. GitHub will instantly pick N reviewers and assign them to the pull request (number of reviewers is configured in the "Code review assignment" settings).
+3. The team will be unassigned.
 
-```
-# Global rule for the whole codebase.
-#  - The empty `iOS-PullAssigner` team will act as a proxy for the PullAssigner bot.
-#    When a new PR arrives, PullAssigner will be triggered, which will then pick members from the iOS-Admin team
+Congratulations, you now have reviewers for your PR and you didn't need to lift a finger.
 
-* @babylonhealth/iOS-PullAssigner
-```
+#### Excluding yourself from auto assignment
 
-Of course, if you created a separate proxy team instead of using our common `iOS-PullAssigner` GitHub proxy team, adapt the content accordingly
+There are some scenarios where you'd like to prevent yourself from being automatically assigned as a reviewer, e.g. holidays. To do this, simply click your avatar in the top right of GitHub, click "Set status" and then tick "Busy". Busy team members will not be automatically assigned as reviewers.
```

Comment:
We should probably share this in `#ios-pedia`

Reply:
Good point! It could be also updated [here](https://github.com/babylonhealth/ios-playbook/blob/master/Cookbook/Technical-Documents/OutOffOfficeRequest.md)